python/lsst/ip/isr/isrTaskLSST.py

```diff
@@ -0,0 +1,1219 @@
+        isCalibration=True,
+    )
+    deferredChargeCalib = cT.PrerequisiteInput(
+        name="cpCtiCalib",
+        doc="Deferred charge/CTI correction dataset.",
+        storageClass="IsrCalib",
+        dimensions=["instrument", "detector"],
+        isCalibration=True,
+    )
+    linearizer = cT.PrerequisiteInput(
+        name='linearizer',
+        storageClass="Linearizer",
+        doc="Linearity correction calibration.",
+        dimensions=["instrument", "detector"],
+        isCalibration=True,
+        minimum=0,  # can fall back to cameraGeom
+    )
+    ptc = cT.PrerequisiteInput(
+        name="ptc",
+        doc="Input Photon Transfer Curve dataset",
+        storageClass="PhotonTransferCurveDataset",
+        dimensions=["instrument", "detector"],
+        isCalibration=True,
+    )
+    crosstalk = cT.PrerequisiteInput(
+        name="crosstalk",
+        doc="Input crosstalk object",
+        storageClass="CrosstalkCalib",
+        dimensions=["instrument", "detector"],
+        isCalibration=True,
+        minimum=0,  # can fall back to cameraGeom
+    )
+    crosstalkSources = cT.PrerequisiteInput(
+        name="isrOverscanCorrected",
+        doc="Overscan corrected input images.",
+        storageClass="Exposure",
+        dimensions=["instrument", "exposure", "detector"],
+        deferLoad=True,
+        multiple=True,
+        lookupFunction=crosstalkSourceLookup,
+        minimum=0,  # not needed for all instruments, no config to control this
+    )
+    defects = cT.PrerequisiteInput(
+        name='defects',
+        doc="Input defect tables.",
+        storageClass="Defects",
+        dimensions=["instrument", "detector"],
+        isCalibration=True,
+    )
+    bfKernel = cT.PrerequisiteInput(
+        name='brighterFatterKernel',
+        doc="Newer complete kernel + gain solutions.",
+        storageClass="BrighterFatterKernel",
+        dimensions=["instrument", "detector"],
+        isCalibration=True,
+        minimum=0,  # can use either bfKernel or newBFKernel
+    )
+    dark = cT.PrerequisiteInput(
+        name='dark',
+        doc="Input dark calibration.",
+        storageClass="ExposureF",
+        dimensions=["instrument", "detector"],
+        isCalibration=True,
+    )
+    outputExposure = cT.Output(
+        name='postISRCCD',
+        doc="Output ISR processed exposure.",
+        storageClass="Exposure",
+        dimensions=["instrument", "exposure", "detector"],
+    )
+    preInterpExposure = cT.Output(
+        name='preInterpISRCCD',
+        doc="Output ISR processed exposure, with pixels left uninterpolated.",
+        storageClass="ExposureF",
+        dimensions=["instrument", "exposure", "detector"],
+    )
+    outputStatistics = cT.Output(
+        name="isrStatistics",
+        doc="Output of additional statistics table.",
+        storageClass="StructuredDataDict",
+        dimensions=["instrument", "exposure", "detector"],
+    )
```

Comment:
I think we also want the rebinned exposure output. https://github.com/lsst/ip_isr/blob/ba5f90d8bcf72f7eb35098846ed0dda4999b39e7/python/lsst/ip/isr/isrTask.py#L263-L274

Reply:
This was updated in the old isr task while I was working on this new one - is it ok if I keep this for DM-40903?

Reply:
If you can copy over the binned exposure stuff that would be great.  But it can also be punted.